gramps/plugins/view/pedigreeview.py

```diff
@@ -784,13 +784,16 @@ def _connect_db_signals(self):
         Connect database signals.
         """
         self._add_db_signal('person-add', self.person_rebuild)
+        # Adding or deleting events will indirectly trigger a person update event
```

Comment:
This comment is slightly confusing here, but you are correct, adding or removing an event will trigger a 'person-update' signal.

Reply:
Where would be the preferred location? Would it be better at the 'event-update' signal or the comment at the top of the function definition?

Reply:
I think that we can just remove the comment.

Reply:
Thanks.  I'll merge this shortly.